testground/benchmark/benchmark/sendtx.py

```diff
@@ -78,15 +80,51 @@ def generate_load(num_accounts, num_txs, global_seq, **kwargs):
                 print("test task failed", e)
 
 
-def prepare_txs(global_seq, num_accounts, num_txs):
+def async_write_tx(account, tx, file_path):
+    raw_tx = account.sign_transaction(tx).rawTransaction
+    with open(file_path, "w") as file:
+        file.write(ujson.dumps([raw_tx.hex()]))
+
+
+def get_txs_foler(outdir: Path, group, global_seq):
+    folder = outdir / TMP_TXS_FOLDER / group / str(global_seq)
+    folder.mkdir(parents=True, exist_ok=True)
+    return folder
+
+
+def prepare_txs(
+    outdir: Path, group, global_seq, num_accounts, num_txs, max_workers=8000
+):
+    folder = get_txs_foler(outdir, group, global_seq)
     accounts = [gen_account(global_seq, i + 1) for i in range(num_accounts)]
-    txs = []
-    for i in range(num_txs):
-        for acct in accounts:
-            txs.append(acct.sign_transaction(test_tx(i)).rawTransaction.hex())
-            if len(txs) % 1000 == 0:
-                print("prepared", len(txs), "txs")
+    with ThreadPoolExecutor(max_workers=max_workers) as executor:
```

Comment:
does it actually improve speed with `8000` workers?

Reply:
generate faster one by one?

Reply:
for cpu bound tasks, the actual parallism won't exceeds the number of cpus that's available, for python there's also the GIL effect, so better to give it a test.

Reply:
keep old way, I get can't start new thread when max_workers > 9k

Reply:
there's no point to start that many threads.